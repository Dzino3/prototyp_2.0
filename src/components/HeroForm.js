import React, { useReducer } from 'react'
import './style/heroStyle.css';

function HeroForm({user}) {
  return (
    <div className='heroDiv'>
        <h1>Welcome, <span>{user.dealer}</span></h1>
    </div>
  )
}

export default HeroForm
