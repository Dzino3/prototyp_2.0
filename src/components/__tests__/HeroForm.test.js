import React from 'react';
import renderer from 'react-test-renderer';
import HeroForm from "../HeroForm.js";

describe(`Rendering HeroForm`, () => {

    test('should render HeroForm with user Hallo42', function () {
        let userX = {
            dealer: "Hallo42"
        };
        let tree = renderer.create(
            <HeroForm user={userX}></HeroForm>
        );

        expect(tree.toJSON()).toMatchSnapshot();
    });

    test('should render HeroForm with empty user', function () {
        let userX = {};
        let tree = renderer.create(
            <HeroForm user={userX}></HeroForm>
        );

        expect(tree.toJSON()).toMatchSnapshot();
    });
})
