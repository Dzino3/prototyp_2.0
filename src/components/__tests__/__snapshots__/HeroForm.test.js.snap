// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering HeroForm should render HeroForm with empty user 1`] = `
Array [
  <div
    class="parallax"
  />,
  <div
    className="heroDiv"
  >
    <h1>
      heroWelcome
       
      <span />
    </h1>
  </div>,
]
`;

exports[`Rendering HeroForm should render HeroForm with user Hallo42 1`] = `
Array [
  <div
    class="parallax"
  />,
  <div
    className="heroDiv"
  >
    <h1>
      heroWelcome
       
      <span>
        Hallo42
      </span>
    </h1>
  </div>,
]
`;
